prow/cluster/velodrome/velodrome_configmap-nginx.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nginx
+  namespace: velodrome
+  labels:
+    app.kubernetes.io/part-of: velodrome
+data:
+  nginx.conf: |
+    server {
+      listen 80 default_server;
+
+      server_name _;
+
+      location / {
+        if ($http_x_forwarded_proto = 'http') {
+            return 301 https://$host$request_uri;
+        }
+        proxy_pass http://130.211.224.127:80/;
```

Comment:
Shouldn't this be `http://velodrome/` to use the k8s service instead of the k8s endpoint that was deleted?

Reply:
This is the public static IP of the velodrome deployment running in the other cluster: https://github.com/kubernetes/test-infra/blob/f421e19a48dc0578eaa937c7ce34927737caa40e/velodrome/config.yaml#L172

Reply:
Does this now look correct? The ingress forwards all traffic to the _velodrome_ service which is associated with a lightweight nginx proxy to redirect http -> https. Otherwise, proxy traffic to the  _velodrome-external_ service which has an endpoint pointed to the static ip of the actual velodrome instance for Istio running in another cluster.